README.md

```diff
@@ -2,6 +2,8 @@
 
 Deploy a single page app on AWS in one command.
 
+> Note: this repository is intended for internal use at Lalilo, and supports only our current deployment needs. We recommend against using it as-is in your production apps as our updates may contain breaking changes and won't be maintained to support other use cases.
+
```

Comment:
Cool ! Do we intend on making the repo private finally ?

Reply:
I guess we can leave it public with that warning, unless you see a reason to make it private?